crates/lib/src/install/baseline.rs

```diff
@@ -36,6 +36,10 @@ use bootc_mount::is_mounted_in_pid1_mountns;
 // This ensures we end up under 512 to be small-sized.
 pub(crate) const BOOTPN_SIZE_MB: u32 = 510;
 pub(crate) const EFIPN_SIZE_MB: u32 = 512;
+/// EFI Partition size for composefs installations
+/// We need more space than ostree as we have UKIs and UKI addons
+/// We might also need to store UKIs for pinned deployments
+pub(crate) const CFS_EFIPN_SIZE_MB: u32 = 1024;
```

Comment:
(Ultimately of course we want this to be more OS controllable, and that thread is adding support for repart.d in to-disk)